assets/styledown.css

```diff
@@ -228,10 +228,12 @@ code.sg {
   content: '\2715';
   top: -1px;
 }
-.sg-expando-contract {
-  margin-top: 20px;
+
+.sg-code {
+  margin: 20px 0;
 }
-/*
- * hover behavior
- */
 
+/* Only hide toggle buttons placed after a hidden code block */
+.sg-code.sg-hidden + .sg-expando {
+  display: none;
+}
```

Comment:
The initial toggle button `.sg-expando-contract` does not control his margin anymore and is hidden when placed after a hidden code block.

Also now a code block `sg-code` will push before and after for the same amount of pixels.